### ISSUE TITLE
Add keyboard shortcuts for tab navigation

### ISSUE THREAD

**Issue by jandubois** [rancher-sandbox/rancher-desktop#2627]
In most apps you can use ⌘1 to ⌘9 to switch from mayor group 1 to 9 (sometimes 9 means last and not 9th), and you can use ⇧⌘] and ⇧⌘[ to cycle forward and backwards through these groups (macOS shortcuts, Windows may use ⌃1 to ⌃9 etc).

In a browser or terminal app the major groups are tabs. In Rancher Desktop the major groups are the top level menu items on the left, so I would expect to use ⌘2 to switch to "Port Forwarding", ⌘3 to "Images" etc.

The same mechanism should work inside the preferences dialog as well, so if I type ⌘, followed by ⌘4 I should be seeing the "Kubernetes" preferences, and then pressing ⇧⌘] should cycle to the "Application" preferences.

The number key shortcuts are strictly based on position, and the corresponding tab may change if we insert additional menu items in the middle.

**Comment by gaktive** [comment]
We may push this out if we're re-evaluating the preferences order.
